nextflow.config

```diff
@@ -157,14 +158,6 @@ profiles {
     }
 }
 
-// Set default registry for Apptainer, Docker, Podman and Singularity independent of -profile
-// Will not be used unless Apptainer / Docker / Podman / Singularity are enabled
-// Set to your registry if you have a mirror of containers
-apptainer.registry   = 'quay.io'
```

Comment:
We could set these to `community.wave.seqera.io` if we want think. For the same reasons we added `quay.io` here in the first place.

Reply:
> community.wave.seqera.io/library/tower-cli:0.9.2--dc544a7e574ab73b

What does the `library` mean in the URI? Will this always be the same for different registries? So do we need:

```
apptainer.registry   = 'community.wave.seqera.io'
docker.registry      = 'community.wave.seqera.io'
podman.registry      = 'community.wave.seqera.io'
singularity.registry = 'community.wave.seqera.io'
```

or this:

```
apptainer.registry   = 'community.wave.seqera.io/library'
docker.registry      = 'community.wave.seqera.io/library'
podman.registry      = 'community.wave.seqera.io/library'
singularity.registry = 'community.wave.seqera.io/library'
```

or none of these based on @pditommaso [comment](https://github.com/seqeralabs/nf-aggregate/pull/36/files#r1617896591)?

Reply:
You don't need to use `container` with Community registry, hence you don't need `registry` either